client/my-sites/stats/stats-redirect/index.tsx

```diff
@@ -32,6 +32,9 @@ const StatsRedirectFlow: React.FC< StatsRedirectFlowProps > = ( { children } ) =
 		isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
 	);
 
+	// TODO: Consolidate permissions checks.
```

Comment:
I am not sure if it could be consolidated with the one in `LoadStatsPage`.

Reply:
Good point! It'll be even better.